src/main/riddl/AllIdsProjections/allIdsProjections.riddl

```diff
@@ -20,7 +20,7 @@ result AllSkus is {skus: SKU*}
 
 context AllIdsProjections is  {
   projection AllTenantIdsView is {
-     fields {
+     record fields {
```

Comment:
I am curious, in most other areas of RIDDL you have a keyword, in this case record, followed by the name of that thing. 

So my question is, did the keyword change to record fields? or is it just record <name>?

In other words, if you changed this line to:
  record foo {
Would it still compile?

If yes, then should we be considering a different or better name for this record definition aside from fields?

Reply:
Its just "record".  I changed the title of this PR :)

Reply:
My bad for the vagueness. Thanks for updating the title! I've changed the 'fields' names to be more descriptive by having the View name and replacing View with Fields.

eg. AllTenantIdsView will have the record AllTenantIdsFields